### PR TITLE
Install specific version

### PR DESCRIPTION
**Pull request by p1nox**
Add an input parameter to bash script installer in order to decide which version to install. If no param is passed, latest version it's going to be installed.

### Usage

* Install old `0.5.0`:
```
wget -c https://raw.githubusercontent.com/davrodpin/mole/master/tools/install.sh -O get_mole.sh
bash get_mole.sh 0.5.0 && rm -f get_mole.sh
```

* Install latest version:
```
wget -c https://raw.githubusercontent.com/davrodpin/mole/master/tools/install.sh -O get_mole.sh
bash get_mole.sh && rm -f get_mole.sh
```

### CODE REVIEW

File: tools/install.sh [hunk 1]
```diff
@@ -42,20 +43,22 @@ fi
...
+	if [ $? -ne 0 ]; then
+		echo -ne "\nThere was an error trying to check what is the latest version of mole.\nPlease try again later.\n"
+		exit 1
+	fi
 fi
 
-filename="mole${latest_version#v}.${os_type}-amd64.tar.gz"
-download_link="https://github.com/${repository}/releases/download/${latest_version}/${filename}"
+filename="mole${target_version#v}.${os_type}-amd64.tar.gz"
+download_link="https://github.com/${repository}/releases/download/${target_version}/${filename}"
```
Comment (davrodpin) [inline tools/install.sh:56]:
```suggestion
download_link="https://github.com/${repository}/releases/download/v${target_version}/${filename}"
```

There is missing `v` at the beginning of the tag version in the download url.

Comment (p1nox) [reply tools/install.sh:56]: Got it, I put it without the `v` expecting the whole version tag, but make more sense to have it there and expect the version number only 👍

Comment (p1nox) [reply tools/install.sh]: PR description updated.

**Comment by davrodpin** [comment]
Welcome and thank you for your contribution to the project, @p1nox!

**Comment by p1nox** [comment]
Awesome! Thank you 😄 👍

**Pull request merged**